BuildYourOwnCanvasSamples/3.single-sign-on/index.html

```diff
@@ -193,8 +193,7 @@
 (async function main() {
 
     // Add your BOT ID below 
-    var BOT_ID = "<BOT ID>";
-    var theURL = "https://powerva.microsoft.com/api/botmanagement/v1/directline/directlinetoken?botId=" + BOT_ID;
+    var theURL = "<Token endpoint URL>"
```

Comment:
```suggestion
    var theURL = "<Token endpoint URL>" // you can find the token URL via the mobile app channel configuration
```

Would it make sense to add where to find this token endpoint url?